src/operator/journal.rs

```diff
@@ -64,10 +74,13 @@ async fn reconcile(journal: Journal, ctx: Context<Data>) -> Result<ReconcilerAct
         .patch_status(&name, &ps, &new_status)
         .await
         .map_err(Error::KubeError)?;
-    let current = match deploys.get(&name).await {
-        Ok(current) => Some(current),
-        Err(kube::Error::Api(e)) => None,
-        _ => None // should actually throw the err
+    return match deploys.get(&name).await {
+        Ok(current) => update(),
+        Err(kube::Error::Api(e)) => create(deploys, journal).await,
+        // TODO(gaocegege): Use error_policy here.
```

Comment:
I think there is no need to use error_policy manually as we have already registered it as the error handler function for the reconcile function, just return the error will.